modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -171,14 +180,13 @@ data:
       remoteWrite:
       - url: "https://remote-write-endpoint.example.com"
         <endpoint_authentication_credentials>
-        <your_write_relabel_configs> <1>
+        <your_write_relabel_configs> #<1>
 ----
 <1> The write relabel configuration settings.
 +
-For `<your_write_relabel_configs>` substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
-+
-The following sample shows how to forward a single metric called `my_metric`:
+For `<your_write_relabel_configs>`, substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint:
```

Comment:
Similar note as above.

Reply:
Hello @maxwelldb I am not sure exactly how did you mean to change this part, I apologise :sweat_smile:  but I understand the issue you see. Would change like [this](https://github.com/openshift/openshift-docs/compare/22a2da2339f9f8780f48fabd0263e3379ca4bcda..850b21a81eb62387e31d7a7549e2b161cb1fe0dc) work?